### PR TITLE
AniCloud 1.0.4

### PR DESCRIPTION
**Pull request by Emijor**
I have changed a few things

### CODE REVIEW

File: websites/A/Anicloud/presence.ts [hunk 1]
```diff
@@ -1,155 +1,239 @@
...
+  
+    video = document.querySelector(
+    "video"
+    );
     if (video !== null) {
       played = video.currentTime != 0;
-      duration = video.duration;
-      current = video.currentTime;
+      timeEnd = video.duration;
+      currentTime = video.currentTime;
```
Comment (SlowLife1661) [inline websites/A/Anicloud/presence.ts:123]:
```suggestion
      timeEnd = video.duration;
      ({currentTime} = video);
```

File: websites/A/Anicloud/presence.ts [hunk 2]
```diff
@@ -1,155 +1,239 @@
...
       paused = video.paused;
     }
     if (played) {
       if (!paused) {
-        const timestamps = presence.getTimestamps(current, duration);
-        presenceData.endTimestamp = timestamps[1];
+        const timestamps = presence.getTimestamps(currentTime, timeEnd);
+        data.endTimestamp = timestamps[1];
```
Comment (SlowLife1661) [inline websites/A/Anicloud/presence.ts:129]:
```suggestion
        [, data.endTimestamp] = timestamps;
```

**Review by Slowlife01 — CHANGES_REQUESTED** [review]
bump version and would be good if you run prettier on your code

**Review by Slowlife01 — DISMISSED** [review]

**Review by Bas950 — CHANGES_REQUESTED** [review]
Upload proof of it working

**Comment by Emijor** [comment]
image


image

image

**Comment by Emijor** [comment]
image

image

**Review by Bas950 — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**